src/libusermode/userhook.hpp

```diff
@@ -245,6 +247,10 @@ struct usermode_cb_registration
     void* extra;
 };
 
+using dll_name_t = std::string;
+using hook_filter_t = std::function<bool(plugin_target_config_entry_t const&)>;
+using wanted_hooks_t = std::map<dll_name_t, std::vector<plugin_target_config_entry_t>>;
```

Comment:
Maybe `std::unordered_multimap` ?

Reply:
We need a structure that allows fast traverse only unique DLL subpathes.

Unordered multimap has no ordering and we need traverse each elements.